src/platforms/unity/native-support.mdx

```diff
@@ -14,14 +14,38 @@ Native support requires configuration using the Sentry Unity Editor window. Supp
 
 Native support is `enabled` by default. The Unity SDK passes the options defined through the Sentry Unity Editor window during build time. For Android, it embeds the [Android SDK](/platforms/android/) inside the Gradle project created by Unity. The [iOS SDK](/platforms/apple/guides/ios/) is embedded inside the generated Xcode project. This means there's no need to use the Unity [built-in crash reporting](https://docs.unity3d.com/ScriptReference/CrashReport.html) functionality.
 
+You can opt out of native support by unchecking `iOS Native Support` and `Android Native Support` in the configuration window.
+
+## Debug Symbols
+
+Sentry requires [debugs information files](platforms/android/data-management/debug-files/) to symbolicate your crash logs. Our Unity SDK does not yet support the automated upload of debug symbols, but since it's using other Sentry SDKs this can be set up manually.
+
+After downloading and installing [sentry-cli](/product/cli/installation/) you can run:
+
+```bash
+sentry-cli --auth-token YOUR_AUTH_TOKEN upload-dif --org ___ORG_SLUG___ --project ___PROJECT_SLUG___ PATH_TO_SYMBOLS
+```
+
 <Note>
 
-Our Unity SDK currently supports native iOS when building for device only. Support for the iOS simulators is planned.
+You can [create the Auth Token here](https://sentry.io/api/).
 
 </Note>
 
-You can opt out of native support by unchecking `iOS Native Support` and `Android Native Support` in the configuration window.
+### Android
 
-## Debug Symbols
+To upload debug symbols for Android all you have to do is to make sure `Create symbols.zip` is enabled in the build settings. Unity will generate them in the same directory as the build output and sentry-cli accepts .zip files. Check out our [Android documentation](/platforms/android/proguard/) for more details.
```

Comment:
We want to avoid language that might imply something is easy to do ("all you have to do"). I know that's not exactly what we're saying, but I just want to be careful. 

```suggestion
To upload debug symbols for Android, just make sure `Create symbols.zip` is enabled in the build settings. Unity will generate them in the same directory as the build output, and sentry-cli accepts .zip files. Check out our [Android documentation](/platforms/android/proguard/) for more details.
```

Reply:
Thanks for pointing that out. I'll keep it in mind in the future.